packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -354,62 +354,66 @@ void main() {
       logs.clear();
     }, variant: KeySimulatorTransitModeVariant.all());
 
-    testWidgets('FocusManager responds to app lifecycle changes.', (WidgetTester tester) async {
-      Future<void> setAppLifecycleState(AppLifecycleState state) async {
-        final ByteData? message = const StringCodec().encodeMessage(state.toString());
-        await TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger
-            .handlePlatformMessage('flutter/lifecycle', message, (_) {});
-      }
-
-      final BuildContext context = await setupWidget(tester);
-      final FocusScopeNode scope = FocusScopeNode(debugLabel: 'Scope');
-      addTearDown(scope.dispose);
-      final FocusAttachment scopeAttachment = scope.attach(context);
-      final FocusNode focusNode = FocusNode(debugLabel: 'Focus Node');
-      addTearDown(focusNode.dispose);
-      final FocusAttachment focusNodeAttachment = focusNode.attach(context);
-      scopeAttachment.reparent(parent: tester.binding.focusManager.rootScope);
-      focusNodeAttachment.reparent(parent: scope);
-      focusNode.requestFocus();
-      await tester.pump();
-      expect(focusNode.hasPrimaryFocus, isTrue);
-
-      await setAppLifecycleState(AppLifecycleState.paused);
-      expect(focusNode.hasPrimaryFocus, isFalse);
-
-      await setAppLifecycleState(AppLifecycleState.resumed);
-      expect(focusNode.hasPrimaryFocus, isTrue);
-    });
-
-    testWidgets('Node is removed completely even if app is paused.', (WidgetTester tester) async {
-      Future<void> setAppLifecycleState(AppLifecycleState state) async {
-        final ByteData? message = const StringCodec().encodeMessage(state.toString());
-        await TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger
-            .handlePlatformMessage('flutter/lifecycle', message, (_) {});
-      }
-
-      final BuildContext context = await setupWidget(tester);
-      final FocusScopeNode scope = FocusScopeNode(debugLabel: 'Scope');
-      addTearDown(scope.dispose);
-      final FocusAttachment scopeAttachment = scope.attach(context);
-      final FocusNode focusNode = FocusNode(debugLabel: 'Focus Node');
-      addTearDown(focusNode.dispose);
-      final FocusAttachment focusNodeAttachment = focusNode.attach(context);
-      scopeAttachment.reparent(parent: tester.binding.focusManager.rootScope);
-      focusNodeAttachment.reparent(parent: scope);
-      focusNode.requestFocus();
-      await tester.pump();
-      expect(focusNode.hasPrimaryFocus, isTrue);
-
-      await setAppLifecycleState(AppLifecycleState.paused);
-      expect(focusNode.hasPrimaryFocus, isFalse);
-
-      focusNodeAttachment.detach();
-      expect(focusNode.hasPrimaryFocus, isFalse);
-
-      await setAppLifecycleState(AppLifecycleState.resumed);
-      expect(focusNode.hasPrimaryFocus, isFalse);
-    });
+    // Skipping the FocusManager app lifecycle tests for now,
```

Comment:
You can uncomment these, they're not failing.

You might want to add a test that makes sure it doesn't do anything on Android.